### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by Rockyraven**
built wishlist page
add navbar footer
add card and final price detai
wishlist
l

### CODE REVIEW

File: Quickstore/pages-styles/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+}
+.remove-cart:hover{
+    background-color: var(--primary-color);
+}
+.final-price-tag{
+    grid-column: 2/3;
+    grid-row: 3/4;
+    max-width: 20rem;
+    margin-right: auto;
+    margin-left: 1rem;
```
Comment (sonishreyas) [inline Quickstore/pages-styles/wishlist.css:31]: you can add this as variables later to keep the styling consistent for margin and padding

File: Quickstore/pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (sonishreyas) [inline Quickstore/pages/wishlist.html:5]: You can add an author tag here

File: Quickstore/pages/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>QuickStore | Wishlist</title>
+
+    <!---------component libaray style-sheet ------>
+    <link rel="stylesheet" href="https://boostweb-ui.netlify.app/component/boostweb.css">
+    
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (sonishreyas) [inline Quickstore/pages/wishlist.html:13]: This can be moved into your main component.css so you don't have to include it in every page

**Review by sonishreyas — APPROVED** [review]
I have kept a few suggestions which you can implement later. Rest it looks good

File: Quickstore/pages/wishlist.html (hunk 3 above)
Comment (Rockyraven) [reply Quickstore/pages/wishlist.html:13]: ok I will consider it

File: Quickstore/pages-styles/wishlist.css (hunk 1 above)
Comment (Rockyraven) [reply Quickstore/pages-styles/wishlist.css:31]: ok i will consider

File: Quickstore/pages/wishlist.html (hunk 2 above)
Comment (Rockyraven) [reply Quickstore/pages/wishlist.html:5]: resolve problem

File: Quickstore/pages-styles/wishlist.css [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    height: fit-content;
+    box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+    padding: 1rem;
```
Comment (Shubhamkr2610) [inline Quickstore/pages-styles/wishlist.css:34]: Suggestion: use color with variable name

**Review by Shubhamkr2610 — APPROVED** [review]
Code is clean. Made the changes that I have suggested rest everything looks good to me. You are good to go .

**Pull request merged**